db/migrate/20180418105215_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
A good practice is to use `null: false` with boolean fields, to avoid three-state booleans -you can read about it here: https://robots.thoughtbot.com/avoid-the-threestate-boolean-problem